frontend/src/components/common-stylings/entity-index.css

```diff
@@ -20,6 +20,8 @@
     color: white;
     background-color: var(--royal-sunset-wafer);
     border-radius: var(--entity-index-border-radius);
+    margin-left: auto;
+    margin-right: auto;
```

Comment:
:o did this fix something? just wondering

Reply:
make it in the center

Reply:
nice!